src/App.jsx

```diff
@@ -19,9 +19,9 @@ export default function App() {
             <Route path="/login">
               <Login />
             </Route>
-            <Route path="/treehouse/members">
```

Comment:
Great job here using the react create context to create a Private route and make the user's info available to your entire app instead of having to pass state around to each file.